raidnearby.py

```diff
@@ -658,6 +680,8 @@ def processRaidImage(self, raidfilename):
 
                 pokemon_id = database.get_raid_pokemon_id(session, gym)
                 end_time = database.get_raid_time(session, gym)
+                if end_time == 0:
```

Comment:
The new end_time is only being sent to webhook and not written to the database?

Reply:
You would probably need to modify one of the database methods (maybe update_raid_mon), or add a new one, and pass the value. I'm only using this code for webhooks so I didn't notice the DB wasn't already being updated